frontend/src/components/LedgerClosedTime.tsx

```diff
@@ -18,9 +18,11 @@ const getLedgerClosedStatus = (closedTime: number) => {
   return ClosedStatus.VERY_SLOW;
 };
 
-export const LedgerClosedTime = ({ closedTime }: { closedTime: number }) => (
+type Props = { closedTime: number; showPrefix?: boolean };
+
+export const LedgerClosedTime = ({ closedTime, showPrefix = true }: Props) => (
   <div className="LedgerClosedTime">
-    {`closed in ${closedTime}s`}{" "}
+    {showPrefix && <span>closed in {closedTime}s</span>}
```

Comment:
Actually, I added the span, because of the selector I was using on CSS to add a spacing between the text and the icon. I changed it a little bit to keep the fragment, and I think it's better now.